lucene/test-framework/src/java/org/apache/lucene/analysis/BaseTokenStreamTestCase.java

```diff
@@ -390,19 +406,19 @@ public static void assertAnalyzesTo(Analyzer a, String input, String[] output, i
   }
 
   public static void assertAnalyzesTo(Analyzer a, String input, String[] output, int startOffsets[], int endOffsets[], String types[], int posIncrements[], int posLengths[], boolean graphOffsetsAreCorrect, byte[][] payloads) throws IOException {
-    assertTokenStreamContents(a.tokenStream("dummy", input), output, startOffsets, endOffsets, types, posIncrements, posLengths, input.length(), null, null, graphOffsetsAreCorrect, payloads);
+    assertTokenStreamContents(a.tokenStream("dummy", input), output, startOffsets, endOffsets, types, posIncrements, posLengths, input.length(), null, null, graphOffsetsAreCorrect, payloads, null);
```

Comment:
Since you put back the original method, I think you can also revert this change?

Reply:
ok